platform/linux-generic/odp_stash.c

```diff
@@ -291,16 +403,21 @@ odp_stash_t odp_stash_create(const char *name, const odp_stash_param_t *param)
 	stash = stash_global->stash[index];
 	memset(stash, 0, sizeof(stash_t));
 
+	/* Set ring function pointers */
 	if (ring_u64) {
-		ring_u64_init(&stash->ring_u64.hdr);
-
-		for (i = 0; i < ring_size; i++)
-			stash->ring_u64.data[i] = 0;
+		stash->ring_fn.u64.init      = strict_ring_u64_init;
+		stash->ring_fn.u64.enq_multi = strict_ring_u64_enq_multi;
+		stash->ring_fn.u64.enq_batch = strict_ring_u64_enq_batch;
```

Comment:
`stash->ring_fn.u64.enq_batch` could probably point directly to `strict_ring_u64_enq_multi` already here and then `strict_ring_u64_enq_batch` becomes unnecessay.

(Same with u32.)

Reply:
Fixed in V2.